SHIR/build.ps1

```diff
@@ -30,6 +30,37 @@ function Install-SHIR() {
     }
 
     Write-Log "SHIR MSI Install Successfully"
+    Write-Log "Will remove C:\SHIR\$MsiFileName"
+    Remove-Item "C:\SHIR\$MsiFileName"
+    Write-Log "Removed C:\SHIR\$MsiFileName"
+}
+
+function Install-MSFT-JDK() {
+    Write-Log "Install the Microsoft OpenJDK in the Windows container"
+
+    Write-Log "Downloading Microsoft OpenJDK version 11.0.19 file"
```

Comment:
I hard coded the version because of the path `$JavaHome = 'C:\Program Files\Microsoft\jdk-11.0.19.7-hotspot'`. I will gladly do this if Azure says I should for this pull request to be merged.

Reply:
I wrote about SHIR's java installation detection in https://github.com/Azure/Azure-Data-Factory-Integration-Runtime-in-Windows-Container/pull/17#pullrequestreview-1575772357. I hope PG does something and provides better documentation.

I have not set `JAVA_HOME` or `PATH` on my VM-based SHIR installations and still Java is found by the SHIR.

Reply:
Interesting, I'd like to try building a container without setting `PATH` nor `JAVA_HOME`, I'll probably do it next week

Reply:
@jikuja I built a container image without setting `PATH` nor `JAVA_HOME` and got this error again: `Type=System.DllNotFoundException,Message=Unable to load DLL 'jvm.dll'`, looks like SHIR in a container needs `PATH` and/or `JAVA_HOME`, for now I will not change this either

Reply:
I will update this part to use

```powershell
Start-Process msiexec.exe -Wait -ArgumentList "/i C:\SHIR\microsoft-jdk-11-windows-x64.msi ADDLOCAL=FeatureMain,FeatureEnvironment,FeatureJarFileRunWith,FeatureJavaHome INSTALLDIR="c:\Program Files\Microsoft\" /quiet"
```